types/google-apps-script/apis/calendar_v3.d.ts

```diff
@@ -507,6 +508,20 @@ declare namespace GoogleAppsScript {
         end?: string | undefined;
         start?: string | undefined;
       }
+      interface WorkingLocationProperties {
+        type?: string|undefined;
+        // The documentation does not specify a type. It gives "any value"
+        homeOffice?: unknown|undefined;
+        customLocation?: {label: string};
+        officeLocation?: OfficeLocation|undefined;
+      }
+      interface OfficeLocation {
+        buildingId?: string|undefined;
+        floorId?: string|undefined;
+        floorSectionId?: string|undefined;
+        deskId?: string|undefined;
+        label?: string|undefined;
+      }
```

Comment:
```suggestion
      interface EventWorkingLocationPropertiesOfficeLocation {
        buildingId?: string | undefined;
        floorId?: string | undefined;
        floorSectionId?: string | undefined;
        deskId?: string | undefined;
        label?: string | undefined;
      }
      interface EventWorkingLocationPropertiesCustomLocation {
        label: string;
      }
      interface EventWorkingLocationProperties {
        type?: string | undefined;
        homeOffice?: object | undefined;
        customLocation?: EventWorkingLocationPropertiesCustomLocation | undefined;
        officeLocation?: EventWorkingLocationPropertiesOfficeLocation | undefined;
      }
```

1. Add space between `|`.
2. Add "Event" prefix to interface names. It is same type names written in type hinting of Google Apps Script Editor.
    1. For example, you can find the function `Calendar.newEventWorkingLocationProperties()` exists. It returns EventWorkingLocationProperties.
    2. `Calendar.newEventWorkingLocationPropertiesOfficeLocation()` and `Calendar.newEventWorkingLocationPropertiesCustomLocation()` also exist.